app/scripts/services/authorization.js

```diff
@@ -52,24 +54,32 @@ angular.module("openshiftConsole")
       var deferred = $q.defer();
       currentProject = projectName;
       var projectRules = cachedRulesByProject.get(projectName);
+      var rulesResource = "selfsubjectrulesreviews";
       if (!projectRules || projectRules.forceRefresh) {
-        Logger.log("AuthorizationService, loading user rules for " + projectName + " project");
-        var object = {kind: "SelfSubjectRulesReview",
-                      apiVersion: "v1"
-                    };
-        DataService.create('selfsubjectrulesreviews', null, object, {namespace: projectName}).then(
-          function(data) {
-            var normalizedData = normalizeRules(data.status.rules);
-            var canUserAddToProject = canAddToProject(data.status.rules);
-            cachedRulesByProject.put(projectName, {rules: normalizedData,
-                                                        canAddToProject: canUserAddToProject,
-                                                        forceRefresh: false,
-                                                        cacheTimestamp: _.now()
-                                                      });
-            deferred.resolve();
-          }, function() {
-            deferred.reject();
-        });
+        // Check if APIserver contains 'selfsubjectrulesreviews' resource. If not switch to permissive mode.
+        if (APIService.apiInfo(rulesResource)) {
+          Logger.log("AuthorizationService, loading user rules for " + projectName + " project");
+          var object = {kind: "SelfSubjectRulesReview",
+                        apiVersion: "v1"
+                      };
+          DataService.create(rulesResource, null, object, {namespace: projectName}).then(
+            function(data) {
+              var normalizedData = normalizeRules(data.status.rules);
+              var canUserAddToProject = canAddToProjectCheck(data.status.rules);
+              cachedRulesByProject.put(projectName, {rules: normalizedData,
+                                                      canAddToProject: canUserAddToProject,
+                                                      forceRefresh: false,
+                                                      cacheTimestamp: _.now()
+                                                    });
+              deferred.resolve();
+            }, function() {
+              deferred.reject();
```

Comment:
@jwforres wasn't sure of that... thought that since when the `selfsubjectrulesreviews` is present in the APIserver then we should be determining if to show the action.